### PR TITLE
feat(BreadcrumbSwitcher): creating a BreadcrumbSwitcher component

### PR DESCRIPTION
**Pull request by glekner**
**What**:
creating a BreadcrumbSwitcher component
screenshot 2018-12-30 15 59 30

**Additional issues**:

### CODE REVIEW

**Comment by ohadlevy** [comment]
bump? :)

**Comment by jeff-phillips-18** [comment]
@patternfly/patternfly-react-ux

**Comment by serenamarie125** [comment]
@glekner can you address a few visual issues here?
1. colors used for hover and selected states seem to be reversed
1. the filter component should be wider - the left side of the filter component should be left aligned with the menu items
1. the left/right margins should match.  right now, the long items bleeds into the margin

**Comment by serenamarie125** [comment]
i think the reference to where this came from should be removed from the storybook, thoughts on that @jeff-phillips-18 ?
screen shot 2019-01-17 at 3 15 33 pm

**Comment by serenamarie125** [comment]
One more thing - our PF design doesn't have the "X" in it, although i think that's super helpful.  Should it be selectable and enabled when nothing is in the text field ? Thoughts @maryclarke @lizsurette @Rohoover

**Comment by glekner** [comment]
* rebased and added a highlighter knob

@serenamarie125 @jeff-phillips-18 can you review please?

**Comment by glekner** [comment]
@serenamarie125 I updated the hover color.
Seeing the active color depends on the pathname so I don't think you can see it in the Storybook. anyway I checked it locally and it is #0088ce :)

File: packages/patternfly-3/patternfly-react/less/breadcrumbswitcher.less [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        position: absolute;
+        top: 7px;
+        right: 10px;
+        z-index: 99;
+      }
+    }
+
+    .breadcrumb-switcher-popover-error {
+      text-indent: 10px;
+      color: #f00;
```
Comment (jeff-phillips-18) [inline packages/patternfly-3/patternfly-react/less/breadcrumbswitcher.less]: Choose from the PF color palette

**Comment by glekner** [comment]
@jeff-phillips-18  @serenamarie125 added functionality test, I guess this pr is ready :)

**Comment by jeff-phillips-18** [comment]
@dlabaj Can you take another look?

**Comment by jeff-phillips-18** [comment]
@glekner Could you rebase please? I believe that should fix the test failure.

**Comment by glekner** [comment]
@jeff-phillips-18 rebased

**Comment by jeff-phillips-18** [comment]
@dlabaj @dgutride Can someone take a look please?

**Comment by glekner** [comment]
@jeff-phillips-18 added fixes from https://github.com/theforeman/foreman/pull/6382/files

**Comment by dlabaj** [comment]
LGTM

**Pull request merged**